src/components/sections-view.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SectionDisplay from './section-display.js';
+import { sectionShape } from '../ui/default-prop-types.js';
+
+function SectionsView({ sections, sectionsContainerId, onAction }) {
+  return (
+    <article data-document-key={sectionsContainerId}>
```

Comment:
We can remove this data attribute, I cannot find any usages.

Reply:
Nice, done! The `sectionsContainerId` prop went away along with it.